state/migrations/0005_user_settings.down.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TEMPORARY TABLE users_backup
```

Comment:
Why a `TEMPORARY TABLE`?

Reply:
I'm not sure :) i stole it from `state/migrations/0002_hash_columns.down.sql`. I can change it to a regular table instead.

Reply:
Ah bad copy pasta on my part :)

Reply:
Fixed in f07c787